components/mysql/actions/delete-row/delete-row.mjs

```diff
@@ -0,0 +1,48 @@
+import mysql from "../../mysql.app.mjs";
+
+export default {
+  key: "mysql-delete-row",
+  name: "Delete Row",
+  description: "Delete an existing row. [See the docs here](https://dev.mysql.com/doc/refman/8.0/en/delete.html)",
+  type: "action",
+  version: "0.0.1",
+  methods: {
+    async deleteRows({
+      table, condition, values = [],
+    }) {
+      const options = {
+        sql: `DELETE FROM \`${table}\` WHERE ${condition}`,
```

Comment:
In my view, we need to sanitize these values to [avoid SQL Injection](https://www.sqlshack.com/sanitizing-inputs-avoiding-security-usability-disasters/).
Maybe you can use a NPM Module like [this one](https://www.npmjs.com/package/sqlstring)

Reply:
Yeah so, I didn't do too much on that regard because from the beggining I was seeing there were already calling `prepared statements` which it would do everything is needed against SQL injection just by using the `mysql2` lib. Look at [this](https://github.com/sidorares/node-mysql2#using-prepared-statements) and [this](https://stackoverflow.com/questions/8263371/how-can-prepared-statements-protect-from-sql-injection-attacks) for a reference.
But I'm actually laking of some validations in `delete-row, execute-query and update-row`

Reply:
Okay, since you are using prepared statements, looks fine. I did some tests and I was not able to inject SQL :)